docs/desktop-wpf/fundamentals/styles-templates-overview.md

```diff
@@ -51,9 +51,58 @@ For more information, see [Create a style for a control](styles-templates-create
 
 ## ControlTemplates
 
-In WPF, the <xref:System.Windows.Controls.ControlTemplate> of a control defines the appearance of the control. You can change the structure and appearance of a control by defining a new <xref:System.Windows.Controls.ControlTemplate> for the control. In many cases, this gives you enough flexibility so that you do not have to write your own custom controls.
+In WPF, the <xref:System.Windows.Controls.ControlTemplate> of a control defines the appearance of the control. You can change the structure and appearance of a control by defining a new <xref:System.Windows.Controls.ControlTemplate> and assigning it to a control. In many cases, templates give you enough flexibility so that you do not have to write your own custom controls.
 
-<!--For more information, see [Create a template for a control](styles-templates-create-apply-template.md).-->
+Each control has a default template assigned to the [Control.Template](xref:System.Windows.Controls.Control.Template) property. The template connects the visual presentation of the control with the control's capabilities. Because you define a template in XAML, you can change the control's appearance without writing any code. Each template is designed for a specific control, such as a <xref:System.Windows.Controls.Button>.
+
+Commonly you declare a template as a resource on the `Resources` section of a XAML file. As with all resources, scoping rules apply.
+
+Control templates are a lot more involved than a style. This is more involved because the control template rewrites the visual appearance of the entire control. A style simply applies property changes to the existing control. However, since the template of a control is applied by setting the [Control.Template](xref:System.Windows.Controls.Control.Template) property, you can use a style to define or set a template.
```

Comment:
```suggestion
Control templates are a lot more involved than a style. This is because the control template rewrites the visual appearance of the entire control, while a style simply applies property changes to the existing control. However, since the template of a control is applied by setting the [Control.Template](xref:System.Windows.Controls.Control.Template) property, you can use a style to define or set a template.
```